.agents/skills/pr-flow/SKILL.md

```
@@ -0,0 +1,69 @@
+  --body "## Summary`n<bullets from receipt>`n`n## Task`n- TaskId: <taskId>"
+```
+
+2. Poll CI checks:
+
+```powershell
+gh pr checks <pr-number> --watch --fail-fast
+```
+
+If checks fail, read output, fix if in scope, push again.
+
+3. Wait for CodeRabbit review (Mercury rule: never merge before review completes):
```

markdownlint-cli2 (0.21.0)

[warning] Ordered list item prefix
Expected: 1; Actual: 3; Style: 1/1/1

(MD029, ol-prefix)